### DISCUSSION TITLE
Ability to edit VN name after importing into Manager

### DISCUSSION THREAD

**Discussion by superange128** [micah686/VnManager#23] [Ideas]
Right now it seems like when you import from vndb, it just uses the original Japanese vndb name, and as far as I see no way to edit it.

It would be nice to be able to edit it to localized names or personal nicknames you have for it if you want, would be sorted by that as well.

**Comment by awadallahfarris** [comment]
+1 

Being able to switch between aliases from VNDB, or preferably letting us edit our own names, would be ideal.
